python/GafferSceneUI/LightEditor.py

```diff
@@ -392,35 +393,55 @@ def __toggleBoolean( self, inspectors, inspections ) :
 		) :
 			return False
 
-		valueData = [ i.value() for i in inspections ]
-		soleValue = sole( v.value if v is not None else False for v in valueData )
+		currentValues = []
 
-		for inspector, paths in inspectors.items() :
-			with Gaffer.Context( self.getContext() ) as context :
-				for path in paths :
-					context["scene:path"] = path
+		# Use a single new value for all plugs.
+		# First we need to find out what the new value would be for each plug in isolation.
+		for inspector, pathInspections in inspectors.items() :
+			for path, inspection in pathInspections.items() :
+				currentValue = inspection.value().value if inspection.value() is not None else None
+
+				if isinstance( inspector, GafferSceneUI.Private.AttributeInspector ) :
+					fullAttributes = self.__settingsNode["in"].fullAttributes( path[:-1] )
+					parentValueData = fullAttributes.get( "light:mute", None )
```

Comment:
We can't hardcode "light:mute" here, because this code is intended to run for any boolean attribute (or parameter).

Reply:
This is still hardcoded to `light:mute`.